backend/benefit/applications/api/v1/serializers.py

```diff
@@ -716,6 +717,24 @@ def get_applicant_terms_in_effect(self, obj):
         else:
             return None
 
+    def get_past_benefit_info(self, obj):
+        aggregated_info = get_past_benefit_info(
```

Comment:
function name is a bit confusing here 😅

Reply:
:-D once you see it, you can't unsee it... I'll change it to something more unambiguous